hphp/runtime/ext/pdo/ext_pdo.php

```diff
@@ -1,6 +1,15 @@
 <?hh
 // generated by idl-to-hni.php
 
+/**
+ * Objects implementing PDOEntity can handle value by column name and meta
+ */
+interface PDOEntity {
```

Comment:
I don't have conversation with anybody who in PHP team. So I don't know what idea on the internals.

I have opened feature request last year. There is no action about this issue from PHP team.

Yes we can move into HH namespace to avoid from hubbub for future situation.

I'm going to move into HH namespace if your decision about this way.